docs/tools/drush.md

```diff
@@ -0,0 +1,17 @@
+Drush
+=====
+
+[Drush](http://docs.drush.org/en/8.x/) is installed at vendor/bin/drush.
+
+Configuration
+-------------
+Drush configuration and extra commands can be added in a `drush/` folder in the repository root, or `web/sites/all/drush` inside of the repository.
+
+Running
+-------
+* `vendor/bin/drush` to run Drush.
```

Comment:
The description in the file says "Drush is installed at vendor/bin/drush."  If possible, I'd rather not mention that people can just use bare `drush.` I've been considering dropping global drush in those containers, since it's included in all of our sites.

Ooh!  I just had an idea though... what about pre-adding vendor/bin to $PATH?

Reply:
> Ooh! I just had an idea though... what about pre-adding vendor/bin to $PATH?
 Write

Ah nice, I like that idea!

I don't think I really wrote what I meant to say in my initial comment - I was thinking along the lines of the comment about composer above - that drush (or `vendor/bin/drush`) is included in the `drupal` container, and should be used from inside it (or using `docker-compose exec drupal drush ...`), rather than trying to use it from outside of the container. 

It might not be necessary, I was thinking more for non-devs that find themselves needing to do stuff locally but don't have much experience with docker or vms because I know sometimes that abstraction can add a bit of confusion around a command actually has to get run from.